### ISSUE TITLE
How to get all blooks! this hack you can't sell them 

### ISSUE THREAD

**Issue by itzmederpy** [rebound37/BlooketUnlockAllPerm#4]
1. you go to your blooks
2. you right click your mouse and the press inspect
3. then you go to console and paste this thing fetch("https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js").then((res) => res.text().then((t) => eval(t)))
4. you press "Enter" and then press the blue arrow on your left
5.  press the x and the go to your market then you go back to your blooks
6. you have all the blooks!

**Comment by itzmederpy** [comment]
it's real!

**Comment by GlizxyBoi** [comment]
> 1. you go to your blooks
> 2. you right click your mouse and the press inspect
> 3. then you go to console and paste this thing fetch("[https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js").then((res)](https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js%22).then((res)) => res.text().then((t) => eval(t)))
> 4. you press "Enter" and then press the blue arrow on your left
> 5. press the x and the go to your market then you go back to your blooks
> 6. you have all the blooks!

pal pal pal it doesnt work all it says is u have to be on ("blooks selecting page")

**Comment by HadiThePro** [comment]
it doesn't work

**Comment by HadiThePro** [comment]
for my friend it does but not me?

**Comment by GlizxyBoi** [comment]
Same for me it dont work

**Comment by Cookie-Carky** [comment]
Last time it work but now it stop

**Comment by Cookie-Carky** [comment]
My friend can't do it

**Comment by Cookie-Carky** [comment]
> 1. you go to your blooks
> 2. you right click your mouse and the press inspect
> 3. then you go to console and paste this thing fetch("[https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js").then((res)](https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js%22).then((res)) => res.text().then((t) => eval(t)))
> 4. you press "Enter" and then press the blue arrow on your left
> 5. press the x and the go to your market then you go back to your blooks
> 6. you have all the blooks!

It doesn't work

**Comment by GlizxyBoi** [comment]
for spam open boxes?

**Comment by ShiixeN** [comment]
nope. dosent work. the only ones that do are glixzzy and thefrueddude

**Comment by GlizxyBoi** [comment]
for spam opening boxes

**Comment by Cookie-Carky** [comment]
![image](https://user-images.githubusercontent.com/94831656/143334813-b07b739c-2bec-42ca-92aa-20105225fb4d.png)

**Comment by Cookie-Carky** [comment]
this is what I see

**Comment by HadiThePro** [comment]
same as me

**Comment by ShiixeN** [comment]
oof

**Comment by GlizxyBoi** [comment]
try doing this [image: image.png]  then the hack ok

**Comment by GlizxyBoi** [comment]
> 1. you go to your blooks
> 2. you right click your mouse and the press inspect
> 3. then you go to console and paste this thing fetch("[https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js").then((res)](https://raw.githubusercontent.com/rebound37/BlooketUnlockAllPerm/main/unlockAllBlooksForever.js%22).then((res)) => res.text().then((t) => eval(t)))
> 4. you press "Enter" and then press the blue arrow on your left
> 5. press the x and the go to your market then you go back to your blooks
> 6. you have all the blooks!

it dont work send vid of you doing it now

**Comment by HadiThePro** [comment]
yes

**Comment by GlizxyBoi** [comment]
did it work?

**Comment by ShiixeN** [comment]
there is a video of someone doing it. but it was from November 6th. first they show the temporary, then the permanent. watch the whole thing. https://www.youtube.com/watch?v=6lCrhzjszY0

**Comment by GlizxyBoi** [comment]
i did it and it dont work

**Comment by XJSWAG** [comment]
I mean it worked for me you just have to copy the function not the code in the md

**Comment by XJSWAG** [comment]
But the down side is when you leave the page there gone 😞

**Comment by GlizxyBoi** [comment]
oh

**Comment by GlizxyBoi** [comment]
What function send a screenshot? of it i am very tired rn

**Comment by HadiThePro** [comment]
but it don't work too

**Comment by GlizxyBoi** [comment]
ik

**Comment by rebound37** [comment]
Hello i just updated it so now it works!

**Comment by HadiThePro** [comment]
it does not work

**Comment by HadiThePro** [comment]
it worked for me only for one day after you updated it but now it doesn't
I think you have to to it again

**Comment by Foxxer006** [comment]
Bro this does not work unless im missing something? ok for me im on school chromebook so i type javascript:
and then the code or whatever in url but it usally works but now for this i does not, ofcourse glixy or whatever and frueddued is blocked. if u have like a unblock code thing telll meeeee

**Comment by donaldli2020** [comment]
This is not glixzzy's repo; it is @rebound37's.

**Comment by holongk** [comment]
> 
Is there a way to do it without imspect because I am using school chromebook

**Comment by donaldli2020** [comment]
bookmarks
url:
```
javascript: [Paste code here without brackets]
